indigo_app/templates/indigo_api/_work_filter_form.html

```diff
@@ -0,0 +1,52 @@
+{% load account %}
+
+<form method="GET" id="work-filter-form">
+  <div class="d-flex">
+
+    <div class="flex-grow-1 mr-2">
+      <input type="text" class="form-control filter-search" name="{{ form.q.name }}" placeholder="Filter by title, year or number" value="{{ form.q.value | default:'' }}">
+    </div>
+
+    <!-- Subtypes filter -->
+    <div class="mr-2">
+      <select class="form-control filter-subtype" name="subtype">
+        {% for option in form.subtype %}
+          {{ option }}
+        {% endfor %}
+      </select>
+    </div>    
+
+    <!-- Stubs filter -->
+    <div class="mr-2">
+      <select class="selectpicker filter-stub" name="stub" data-style="btn-outline-secondary" data-width="auto">
+        <option value="excl">Exclude stubs</option>
+        <option value="only">Only stubs</option>
+        <option value="all">Everything</option>
+      </select>
+    </div>
+
+    <!-- Status filter -->
+    <div class="mr-2">
+      <select class="selectpicker notooltip" multiple name="status" data-style="btn-outline-secondary">
+        <option value="published" {% if 'published' in form.status.value %} selected {% endif %} data-content="<i class='fas fa-circle fa-fw published'></i> Published">Published</option>
+        <option value="draft" {% if 'draft' in form.status.value %} selected {% endif %} data-content="<i class='fas fa-circle fa-fw draft'></i> Draft">Draft</option>
+      </select>
+    </div>  
+
+    <!-- Time updated filter -->
+    <div class="mr-2">
+      <select class="form-control sortby" name="sortby">
+        <option value="-updated_at" selected>Most recently updated</option>
```

Comment:
Also need to remember which of these is chosen. Probably can just iterate over the options.